src/main/java/com/internship/frejaeidjmeterplugin/jmeter/sample/AuthSample.java

```diff
@@ -16,41 +11,37 @@
 import org.apache.jmeter.samplers.Entry;
 import org.apache.jmeter.samplers.SampleResult;
 
-/**
- *
- * @author Ivan
- */
 public class AuthSample extends AbstractSampler {
 
-    private static AuthSample instance;
-    
-    private  AuthSample() {
+    private AuthenticationService initRequest;
+
+    public AuthSample() {
+
     }
 
     @Override
     public SampleResult sample(Entry entry) {
-        return null;
+         SampleResult sr = new SampleResult();
+        try {
+            String reference = sendInitAuthenticate("aleksandar.markovic@verisec.com", MinRegistrationLevel.BASIC);
+            AuthenticationResult ar = getResultsInitAuthenticate(reference);
+            sr.setResponseMessage(ar.getStatus()+"");
+            sr.setSampleLabel("Freja eID Test");
+            
+        } catch (FrejaEidClientInternalException | FrejaEidException | FrejaEidClientPollingException ex) {
+            Logger.getLogger(AuthSample.class.getName()).log(Level.SEVERE, null, ex);
+        }finally{
+            return sr;
+        }
     }
 
-    public static AuthSample getInstance() {
-        if (instance == null){
-            return new AuthSample();
-        }
-        return instance;
+    public String sendInitAuthenticate(String email, MinRegistrationLevel registrationLevel) throws FrejaEidClientInternalException, FrejaEidException, FrejaEidClientPollingException {
```

Comment:
I don't think this method is necesseary, we can directly call authService.initAuthRequest